src/flex-lua-table.cpp

```diff
@@ -202,10 +203,12 @@ static std::size_t idx_from_userdata(lua_State *lua_state, int idx,
     return eo;
 }
 
-static void parse_and_set_expire_options(lua_State *lua_state,
-                                         flex_table_column_t *column,
-                                         std::size_t expire_outputs_size,
-                                         bool append_mode)
+} // anonymous namespace
```

Comment:
Too early? The functions below were all static.

Reply:
I have fixed this and rebased to current master.